woocommerce-gateway-amazon-payments-advanced.php

```diff
@@ -207,19 +207,20 @@ public function init_gateway() {
 		include_once $this->includes_path . 'class-wc-gateway-amazon-payments-advanced-privacy.php';
 
 		$subscriptions_installed = class_exists( 'WC_Subscriptions_Order' ) && function_exists( 'wcs_create_renewal_order' );
-		$subscriptions_enabled   = empty( $this->settings['subscriptions_enabled'] ) || 'yes' === $this->settings['subscriptions_enabled'];
```

Comment:
I removed this to avoid having settings calls on the main plugin class. But class loading itself would not be a huge hit in performance (tho it is something). Will likely add a note to check for this.